src/raft/raft.rs

```diff
@@ -513,7 +513,7 @@ impl<T: Storage> Raft<T> {
             return;
         }
         self.election_elapsed += 1;
-        if self.is_election_timeout() {
+        if self.pass_election_timeout() {
```

Comment:
I feel the name `pass_election_timeout` has a strong Chinese taste.
`timeout` has means something pass timeline already